### PR TITLE
fix: set career teaser to 100% width on mobile

### LINKED ISSUE #355: Design: Career blocks
My idea was to display the blocks with icons in sliders. Georgios decided (which is fine for me) to display them all below each other. Problem then: The cards/elements don't have full width on mobile, instead they only occupy like 2/3 of the page. Please make mobile cards full width *if* they are not used in sliders.

### PR DESCRIPTION
**Pull request by hofmann-felix**
Resolves: https://github.com/satellytes/satellytes.com/issues/355
Preview URL: https://satellytescommain21751-fixcareerteaserwidth.gtsb.io/career/

**What's included?**
- A single Teaser on the career page has a width of 100% now

Before: 
Bildschirmfoto 2022-02-08 um 12 18 22

Now: 
Bildschirmfoto 2022-02-08 um 12 19 03

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain21751-fixcareerteaserwidth.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/eb8683d4-5a1c-49e6-8ccc-d7a9ffc250f2/details)

:clock1: Build time: **1m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 67   |
| Accessibility |   :green_heart: 98   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 79   |

:link: [View full report](https://build-eb8683d4-5a1c-49e6-8ccc-d7a9ffc250f2.gtsb.io/reports/lighthouse/index.html)

File: src/components/pages/career/culture.tsx [hunk 1]
```diff
@@ -17,7 +17,12 @@ const TeaserGrid = styled.div`
   gap: 24px;
 
   justify-items: stretch;
-  grid-template-columns: repeat(auto-fit, 250px);
+  grid-template-columns: 1fr;
```
Comment (georgiee) [inline src/components/pages/career/culture.tsx:20]: 👍

**Review by georgiee — APPROVED** [review]
Looks much better, thank you!

**Comment by georgiee** [comment]
I enabled auto-merge on behave of Felix as I spoke to Eric that it was already reviewed by him👍

**Pull request merged**